### ISSUE TITLE
Status of this reimplementation?

### ISSUE THREAD

**Issue by neuromancer** [0xc0de/BladeRemake#1]
I was wondering which is the status of this re-implementation of Blade of Darkness. Is it Linux support planned?

**Comment by Constantine7373** [comment]
Why work is stopped? nedd any help or donate? 

Textures can be upscaled by neuronet. if models easy to export any curent 3D Editor, i can increase detalisation( i have some Blender experience).

**Comment by Sryml** [comment]
@Constantine7373 
> Textures can be upscaled by neuronet. if models easy to export any curent 3D Editor, i can increase detalisation( i have some Blender experience).

Yes, all textures currently have HD versions. They are processed by "Topaz Gigapixel AI" and shared on the MODDB website.
All model files have been decrypted, check the [forum topic](https://arokhslair.net/forum/viewtopic.php?f=29&t=4438&start=45#p34297).
